compiler/bindings/python/IREECompilerDialectsModule.cpp

```diff
@@ -352,7 +352,49 @@ PYBIND11_MODULE(_ireeCompilerDialects, m) {
           "cls"_a, "value"_a, "ctx"_a = py::none(),
           "Gets an #iree_gpu.lowering_config from parameters.")
       .def_property_readonly("attributes",
-                             ireeGPULoweringConfigAttrGetAttributes);
+                             ireeGPULoweringConfigAttrGetAttributes)
+      .def_property_readonly("workgroup_tile_sizes",
+                             [](MlirAttribute self) -> std::vector<int64_t> {
+                               auto tilesizes =
+                                   ireeGPULoweringConfigAttrGetTileSizes(self);
+                               return {tilesizes.workgroupTileSizes,
+                                       tilesizes.workgroupTileSizes +
+                                           tilesizes.numWorkgroupTileSizes};
+                             })
+      .def_property_readonly("reduction_tile_sizes",
+                             [](MlirAttribute self) -> std::vector<int64_t> {
+                               auto tilesizes =
+                                   ireeGPULoweringConfigAttrGetTileSizes(self);
+                               return {tilesizes.reductionTileSizes,
+                                       tilesizes.reductionTileSizes +
+                                           tilesizes.numReductionTileSizes};
+                             })
+      .def_property_readonly(
+          "subgroup_count",
```

Comment:
To me it's not clear how these map to mnk dimensions. Maybe call it `subgroup_count_mn`?